eng/dockerfile-templates/runtime/Dockerfile

```diff
@@ -0,0 +1,53 @@
+# escape=`
+
+FROM mcr.microsoft.com/windows/servercore:{{OS_VERSION_NUMBER}}-amd64
+
+ENV {{
+if OS_VERSION_NUMBER != "ltsc2019"
+:`
+    # Enable detection of running in a container
+    DOTNET_RUNNING_IN_CONTAINER=true `
+    }}COMPLUS_NGenProtectedProcess_FeatureEnabled=0
+
+RUN `
+{{if PRODUCT_VERSION = "3.5"
+:    # Install .NET Fx 3.5
+    curl -fSLo microsoft-windows-netfx3.zip https://dotnetbinaries.blob.core.windows.net/dockerassets/microsoft-windows-netfx3-{{OS_VERSION_NUMBER}}.zip `
+    && tar -zxf microsoft-windows-netfx3.zip `
+    && del /F /Q microsoft-windows-netfx3.zip `
+    && DISM /Online /Quiet /Add-Package /PackagePath:.\microsoft-windows-netfx3-ondemand-package~31bf3856ad364e35~amd64~~.cab `
+    && del microsoft-windows-netfx3-ondemand-package~31bf3856ad364e35~amd64~~.cab `
+    && powershell Remove-Item -Force -Recurse ${Env:TEMP}\* `
+    `
+^elif OS_VERSION_NUMBER = "ltsc2019" && PRODUCT_VERSION = "4.8"
+:    # Install .NET Fx 4.8
+    curl -fSLo dotnet-framework-installer.exe https://download.visualstudio.microsoft.com/download/pr/7afca223-55d2-470a-8edc-6a1739ae3252/abd170b4b0ec15ad0222a809b761a036/ndp48-x86-x64-allos-enu.exe `
+    && .\dotnet-framework-installer.exe /q `
+    && del .\dotnet-framework-installer.exe `
+    && powershell Remove-Item -Force -Recurse ${Env:TEMP}\* `
+    `
+}}{{if VARIABLES[cat("kb|", OS_VERSION_NUMBER)] != void
```

Comment:
This condition supports new windows versions where there is not latest patch to apply. This means there are hopefully no template changes needed when new versions of Windows are released or for the first servicing of the new versions.